src/service/lib/sequelize.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const Sequelize = require(`sequelize`);
+const {DB_NAME, DB_USER, DB_PASSWORD, DB_HOST, DB_PORT} = process.env;
+
+const somethingIsNotDefined = [DB_NAME, DB_USER, DB_PASSWORD, DB_HOST, DB_PORT]
+  .some((param) => param === undefined);
+
+if (somethingIsNotDefined) {
+  throw new Error(`One or more environmental variables are not defined`);
+}
+
+module.exports = new Sequelize(
+    DB_NAME, DB_USER, DB_PASSWORD, {
+      host: DB_HOST,
+      port: DB_PORT,
+      dialect: `postgres`,
+      pool: {
+        max: 5,
+        min: 0,
+        acquire: 10000,
+        idle: 10000
+      }
+    }
```

Comment:
лучше следовать одной конвенции (или строка или столбик)
```suggestion
    DB_NAME, 
    DB_USER, 
    DB_PASSWORD, 
    {
      host: DB_HOST,
      port: DB_PORT,
      dialect: `postgres`,
      pool: {
        max: 5,
        min: 0,
        acquire: 10000,
        idle: 10000
      }
    }
```